resources/js/src/app/components/itemList/filter/ItemColorTiles.js

```diff
@@ -19,6 +19,14 @@ Vue.component("item-color-tiles", {
         })
     },
 
+    mounted() {
+        this.$nextTick(function () {
+            $(function() {
```

Comment:
Do we actually need the `$(function() {...` wrapper, now that we use `mounted`?

Reply:
Apparently not, works well without it.